src/middleware.ts

```diff
@@ -13,7 +13,6 @@ const intlMiddleware = createMiddleware({
 export default authMiddleware({
   publicRoutes: (req: NextRequest) =>
     !req.nextUrl.pathname.includes('/dashboard'),
-  ignoredRoutes: ['/api/guestbook'],
   // By default, the middleware will return a 401 response for all routes `/api/*` when the user is signed out.
```

Comment:
Do you think you can also remove the comment below? it is not relevant anymore.

Thanks in advance.

Reply:
`// By default, the middleware will return a 401 response for all routes `/api/*` when the user is signed out.
// But, for `/api/guestbook`, we want unauthenticated users to be able to access it.`

Reply:
Removed.